### PR TITLE
[KYUUBI #1076] Replace umbrella.md with umbrella.yml

### LINKED ISSUE #1076: [FEATURE] Replace umbrella.md with umbrella.yml
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

As the title says.

### Motivation

_No response_

### Describe the solution

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by byyue**
### _Why are the changes needed?_

Close #1076.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/umbrella.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Umbrella
+title: "[Umbrella] "
+description: An umbrella issue with multiple sub-tasks
+labels: [ "feature" ]
```
Comment (timothy65535) [inline .github/ISSUE_TEMPLATE/umbrella.yml:21]: `labels: [ "umbrella" ]` more resonable, because for serious bug also needs sub-tasks to fix.

Comment (byyue) [reply .github/ISSUE_TEMPLATE/umbrella.yml]: Updated.

File: .github/ISSUE_TEMPLATE/umbrella.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        - label: >
+            I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues) and found no similar
+            issues.
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Describe the proposal
+      description: >
+        Kindly also put the related link here if it is a [KPIP](https://kyuubi.apache.org/improvement-proposals.html).
```
Comment (timothy65535) [inline .github/ISSUE_TEMPLATE/umbrella.yml:46]: `Kindly also put the related link` to `Please put the related link here`

File: .github/ISSUE_TEMPLATE/umbrella.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      placeholder: >
+        Please describe the content of the proposal clearly and concisely.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Task list
+      description: >
+        For more details, kindly refer to [github docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
```
Comment (timothy65535) [inline .github/ISSUE_TEMPLATE/umbrella.yml:56]: `kindly` to `please`

**Comment by timothy65535** [comment]
LGTM, left few comments.

File: .github/ISSUE_TEMPLATE/umbrella.yml (hunk 3 above)
Comment (byyue) [reply .github/ISSUE_TEMPLATE/umbrella.yml:56]: They are synonyms actually. Updated anyway.

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**